spec/requests/api/parent_objects_spec.rb

```diff
@@ -45,4 +49,14 @@
       expect(response.body).to eq("{\"title\":\"Parent Object is restricted.\"}")
     end
   end
+
+  # rubocop:disable Metrics/LineLength
+  describe 'GET metadata from parent object' do
+    it 'displays objects metadata' do
+      ParentObject.create! valid_attributes
+      get "/api/parent/#{valid_attributes[:oid]}"
+      expect(response.body).to match("[{\"dcs\":{\"oid\":\"2004628\",\"visibility\":\"Public\",\"metadata_source\":\"ils\",\"bib\":\"123\",...tp://localhost:3000/manifests/2004628\"},\"metadata\":{\"oid\":\"12345\",\"uri\":\"/uri_example\"}}]")
```

Comment:
"metadata_source\":\"ils\": should be "metadata_source\":\"Ladybird\"  (after updated the metadata_source in the controller)

Reply:
Thanks, Maggie. Changes have been pushed to this PR :)